python/ray/_private/ray_option_utils.py

```diff
@@ -169,9 +169,10 @@ def issubclass_safe(obj: Any, cls_: type) -> bool:
         (int, str, type(None)),
         lambda x: None
         if (x is None or x == "dynamic" or x == "streaming" or x >= 0)
-        else "The keyword 'num_returns' only accepts None, a non-negative integer, or "
-        '"dynamic" (for generators)',
-        default_value=1,
+        else "Default None. The keyword 'num_returns' only accepts None, "
+        "a non-negative integer, or "
+        '"dynamic" (for generators). "dynamic" is deprecated from Ray 2.7.',
```

Comment:
```suggestion
        '"streaming" (for generators). "dynamic" is deprecated from Ray 2.7.',
```
?

Reply:
why the default value is not 1 but None, what does None mean here?

Reply:
Let me update the comment. None basically means default. 

Default behavior is 1 for regular task, and streaming for a generator task

Reply:
updated the comment. Please take a look and lmk what you think